### PR TITLE
Bump documentation theme version

### PR DESCRIPTION
**Pull request by ADubhlaoich**
### Proposed changes

Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to
that issue here in this description (not in the title of the PR).

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [ ] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/go.sum [hunk 1]
```diff
@@ -1,2 +1,4 @@
 github.com/nginxinc/nginx-hugo-theme v0.41.8 h1:l9Lsl9NMaTmNCSMa/W9sdqwhlKFxfb+6ue6w6+bsMC0=
```
Comment (j1m-ryan) [inline docs/go.sum:1]: I think if you do a `go mod tidy` it will get rid of the old dependency

Comment (ADubhlaoich) [reply docs/go.sum:1]: I'll give it a try: I genuinely know nothing about Go tooling outside of my incidental interactions due to Hugo.

**Review by j1m-ryan — APPROVED** [review]

File: docs/Makefile [hunk 2]
```diff
@@ -4,7 +4,7 @@ HUGO_IMG?=hugomods/hugo:0.115.3
 
 THEME_MODULE = github.com/nginxinc/nginx-hugo-theme
 ## Pulls the current theme version from the Netlify settings
```
Comment (pdabelf5) [inline docs/Makefile:6]: Is this still correct?

Comment (ADubhlaoich) [reply docs/Makefile:6]: No, but fixing it is outside the scope of the PR.

The Makefile and README are due an update, which will be handled  separately.

**Review by pdabelf5 — APPROVED** [review]

**Review by jjngx — APPROVED** [review]

**Review by Jcahilltorre — APPROVED** [review]

**Pull request merged**